index.html

```diff
@@ -3,20 +3,11 @@
 <html>
 <head>
 	<title>ES6 Classes</title>
-	<script src="./dist/canvanoid.bundle.js"></script> 
+	
 </head>
 <body>
 	<canvas id="canvas"></canvas>
-	<script> //"babel src -d dist",
-		window.requestAnimationFrame = window.requestAnimationFrame || window.webkitRequestAnimationFrame || 
-									window.msRequestAnimationFrame || window.mozRequestAnimationFrame;
-					
-		var canvas = document.getElementById('canvas')
-		canvas.width = window.innerWidth - 100; 
-		canvas.height = window.innerHeight - 100; 
 
-		var game = new Canvanoid(canvas); 	
-		game.start();
-	</script>
+	<script src="./dist/canvanoid.bundle.js"></script> 
```

Comment:
there is a plugin in webpack that already injects this in the html

 new HtmlWebpackPlugin({
      filename: 'index.html', //Name of file in ./dist/
      template: 'index.html', //Name of template in ./src
			hash: true
    })



Sample:

https://github.com/Lemoncode/webpack-1.x-by-sample/tree/master/00%20Intro/03%20Output